README.md

```diff
@@ -8,157 +8,155 @@
+- [Compatibility](#compatibility)
+- [Debugging](#debugging)
+- [Additional resources](#additional-resources)
+  - [Edge concepts](#edge-concepts)
+  - [CLI](#cli)
+  - [Deploying Edge](#deploying-edge)
+  - [Migrating from Unleash Proxy](#migrating-from-unleash-proxy)
+  - [Performance benchmarking](#performance-benchmarking)
+  - [Contribution and development guide](#contribution-and-development-guide)
 
-Unleash Edge is the successor to the [Unleash Proxy](https://docs.getunleash.io/how-to/how-to-run-the-unleash-proxy).
 
-Unleash Edge sits between the Unleash API and your SDKs and provides a cached read-replica of your Unleash instance.
-This means you can scale up your Unleash instance to thousands of connected SDKs without increasing the number of
-requests you make to your Unleash instance.
+> Availability: Unleash v4.15+.
 
-Unleash Edge offers two important features:
+## Overview
 
-- **Performance**: Unleash Edge caches in memory and can run close to your end-users. A single instance can handle tens
-  to hundreds of thousands of requests per second.
-- **Resilience**: Unleash Edge is designed to survive restarts and operate properly even if you lose connection to your
-  Unleash server.
+Unleash Edge is a fast and lightweight proxy layer between your Unleash API and SDKs. It acts as a read replica of your Unleash instance and is designed to help you scale Unleash. It allows you to support thousands of connected SDKs without increasing the number of requests you make to your Unleash instance.
 
-Unleash Edge is built to help you scale Unleash.
+Edge supports both client-side and server-side SDKs and has multi-environment and project awareness. You can daisy-chain Edge instances to support more complex setups, such as multi-cloud deployments.
 
-* If you're looking for the easiest way to connect your client SDKs you can check out
-  our [Frontend API](https://docs.getunleash.io/reference/front-end-api).
-* If you're looking to learn how to scale your own feature flag system why not check out our recommendations for
-  building and scaling [feature flags](https://docs.getunleash.io/topics/feature-flags/feature-flag-best-practices)
+Key features:
 
-## Migrating to Edge from the Proxy
+- **Performance**: Edge uses in-memory caching and can run close to your end-users. A single instance can handle tens to hundreds of thousands of requests per second.
+- **Resilience**: Edge is designed to survive restarts and maintain functionality even if you lose connection to your Unleash server.
+- **Security**: Edge supports frontend applications without exposing sensitive data to end-users or to Unleash.
 
-For more info on migrating, check out the [migration guide](./migration-guide.md) that details the differences between
-Edge and the Proxy and how to achieve similar behavior in Edge.
+You can run Edge in two different modes: **edge** or **offline**. To learn about the different modes and other Edge concepts, visit [Concepts](./docs/concepts.md).
```

Comment:
I think this is fine but to arm you with more information so you can make decisions - offline mode is almost never what you want, it can be useful for automated testing scenarios when you're trying to mock out Unleash in integration tests but in production you want "edge" mode every time

Reply:
The reason why I wanted to introduce modes here is because we refer to 'edge mode' later on. But we could also just go for something like:

You can run Edge in different modes; visit [Concepts](./docs/concepts.md) to learn more.

Feels a bit more mysterious, but I think it works if we don't want to mention the offline mode.